model/fn-execution/src/main/resources/org/apache/beam/model/fnexecution/v1/standard_coders.yaml

```diff
@@ -152,6 +152,20 @@ examples:
 
 ---
 
+coder:
+  urn: "beam:coder:timer:v1"
+  components: [{urn: "beam:coder:bytes:v1"}]
+examples:
+  "\0\0\0\0\0\0\0\0\u0003abc": {timestamp: -9223372036854775808, payload: abc}
+  "\x7fÿÿÿÿÿÿ\x01\u0003abc": {timestamp: -255, payload: abc}
+  "\x7fÿÿÿÿÿÿÿ\u0003abc": {timestamp: -1, payload: abc}
+  "\x80\0\0\0\0\0\0\0\u0003abc": {timestamp: 0, payload: abc}
+  "\x80\0\0\0\0\0\0\x01\u0003abc": {timestamp: 1, payload: abc}
+  "\x80\0\0\0\0\0\x01\0\u0003abc": {timestamp: 256, payload: abc}
+  "ÿÿÿÿÿÿÿÿ\u0003abc": {timestamp: 9223372036854775807, payload: abc}
+
+---
+
```

Comment:
Out of curiosity, is there a specific reason to place this coder definition here? I initially assumed the coders were sorted in alphabetical order, but it seems to be arbitrary.

Reply:
no specific reason